routes/index.js

```diff
@@ -47,11 +47,6 @@ function getConfig(req, res){
   res.send(cfg);
 }
 
-function getTotals(req, res){  
-
-  res.send("1000000000");
-}
-
 function getCirculating(req, res){  
 
   res.send("500000000");
```

Comment:
Circulating should be made dynamic also: `totalSupply - 500000000`

Reply:
Or the actual current balance of the initial alloc account(s) instead of the fixed value `500000000`.

Reply:
how to get the actual current balance of the initial alloc account? is this available via API?

Reply:
Perhaps it would be clearest to have a separate function `getLocked()` which fetches this account's balance directly for mainnet: https://explorer.gochain.io/addr/0xf75b6e2d2d69da07f2940e239e25229350f8103f and returns 0 otherwise.

Reply:
only one account?(then I suppose we should set it as env variable)
Maybe better to make separate RPC method in gochain?

Reply:
The funds are not actually 'locked' in any meaningful way other than that we promise to hold them, and it only applies to mainnet, so I think it's fine to have it coded in to explorer and I hope it is as simple as that single initial allocation account, but there may be others to include as well.